tests/helpers/core-manager.js

```diff
@@ -40,3 +42,37 @@ export function replicate(cm1, cm2) {
     ])
   }
 }
+
+export async function waitForCores (coreManager, keys) {
```

Comment:
I moved these functions into the core-manager helpers file. Haven't imported them into the core-manager tests yet but can do that.